tests/color_refinement.py

```diff
@@ -34,15 +35,16 @@
 files = ['colorref_smallexample_4_7', 'colorref_smallexample_6_15', 'colorref_smallexample_2_49']
 for i_file in range(0,len(files)):
     file = files[i_file]
-    filename = '../test_graphs/color_refinement/' + file + '.grl'
+    filename = '/test_graphs/color_refinement/' + file + '.grl'
     graphs = load_graph_list(filename)
 
     for i in range(0, len(graphs) - 1):
         for j in range(i + 1, len(graphs)):
             G = graphs[i] + graphs[j]
 
             start = time()
-            color_refinement(degree_color_initialization(G))
+            remove_twins(G)
+            color_refinement(twins_color_initialization(G))
```

Comment:
Lijkt mij beter dat de methode twins_color_intitialization tegen degree_color_initialization wordt getest.
Of in een losse test.

Reply:
Oh dit was ook niet mijn bedoeling, had het weer willen verwijderen, mijn fout!